jcl/src/openj9.dtfj/share/classes/com/ibm/jvm/j9/dump/extract/Main.java

```diff
@@ -338,14 +335,10 @@ public static void main(String[] args) {
 
 		Main dumper = new Main(dumpName, virtualRootDirectory, verbose, throwExceptions, disableBuildIdCheck, excludeCoreFile);
 
-		if (interactive) {
-			dumper.runInteractive();
-		} else {
 			if (zip) {
 				dumper.runZip(outputName);
 			}
 			report("jextract complete."); //$NON-NLS-1$
```

Comment:
Please also remove `zip` (which can only ever be `true`), and correct the indentation here.